### ISSUE TITLE
sp_ineachdb - IIF statement introduced errors in SS2008 & 2008R2

### ISSUE THREAD

**Issue by sm8680** [BrentOzarULTD/SQL-Server-First-Responder-Kit#2744]
The IIF statement introduced recently in the code errors SQL Server 2008 and 2008R2 due to the IIF statement being introduced in SQL Server 2012. Now I know a lot of us including myself would only like to have the latest 3 versions of SQL Server in production. However for large organizations that's not always the case and unfortunately in those cases we have versions of SQL Server that aren't supported.

Thanks
Steven

**Comment by BrentOzar** [comment]
OK, cool. Are you volunteering to contribute the code necessary to get it to work on 2008/2008R2?

Our alternatives are:
* You continue to use the prior version with 2008/2008R2 compatibility
* You contribute code to get this one to work on 2008/2008R2

I don't use 2008/2008R2 anymore, and unfortunately open source isn't free development consulting, so unfortunately this isn't something we can just assign to others.

**Comment by sm8680** [comment]
Sure let me look into this.

**Comment by sm8680** [comment]
Brent I just did a pull.  So you now have 2 pulls on this sproc.  The older pull fixes the 3 case sensitivity issues on variable @NoSpaces.

The newer pull contains the case sensitivity fix plus the following.  

For SQL Server compatibility to SS2008 and SS2008R2. 
4 changes for IIF statements to CASE statements.  
1 change for CONCAT statement to a ISNULL(C.Name,'') + ...  

Tested on SS 2008, 2008R2, 2012, 2014, 2017, 2019 and case sensitive systems.
